### PR TITLE
Setup sponsorships

### PR DESCRIPTION
**Pull request by BasThomas**
References #410.

This is a work in progress and should not yet be merged.

### CODE REVIEW

**Comment by jessesquires** [comment]
LGTM so far

**Comment by jessesquires** [comment]
Still todo:

- On `sponsorship.md` add a method to contact you
- Update the mailing list sign up form

File: subscribe.html [hunk 1]
```diff
@@ -10,14 +10,6 @@ <h2 class="text-center">{{ page.title }}</h2>
...
-        <h3>On hiatus <small class="text-danger"><i>04 Jan 2018</i></small></h3>
-            <p>Our mailing list is currently paused. We might resume emails in the future, so you can still subscribe if you'd like.
-                Until then, you can <a class="alert-link" href="https://twitter.com/swiftlybrief">follow us on Twitter</a>.
-                Thank you for your support!
-            </p>
-        </div>
-
     <!-- Begin MailChimp Signup Form -->
     <div id="mc_embed_signup">
         <form action="//github.us12.list-manage.com/subscribe/post?u=5230bf0236a0adb19995a2eb4&amp;id=0d4b4bbe45"
```
Comment (jessesquires) [inline subscribe.html]: you'll need to update this for your mailchimp account

Comment (BasThomas) [reply subscribe.html]: Done!

**Comment by BasThomas** [comment]
> Still todo:
> 
> * On `sponsorship.md` add a method to contact you
> * Update the mailing list sign up form

These are done :)

**Pull request merged**